### DISCUSSION TITLE
V1.2 Emissions factors by summary code

### DISCUSSION THREAD

**Discussion by smcnichols94** [USEPA/supply-chain-factors#52] [Q&A]
Hello,

I am wondering if there is a version of the V1.2 emissions factors by summary code like there was provided last year? If not, what is the recommendation for manipulating the factors to get to a more high-level summary emissions factor?

Thanks!
Susan

**Comment by bl-young** [comment]
Correct this version does not include a set of summary level factors. The Reference USEEIO code provided along with the factors aligns very closely with the BEA detail schema. We'd recommend using those detailed codes if possible. 

Most of the BEA files include a mapping of detail to summary codes to help identify which may be most appropriate.

If you really need summary level factors you could perhaps consider aggregating the detail level factors and weighting them by industry output (e.g., from the [Make table](https://apps.bea.gov/industry/xls/io-annual/IOMake_Before_Redefinitions_Det.xlsx))
